src/main/java/mate/academy/internetshop/service/BucketService.java

```diff
@@ -17,5 +18,5 @@
 
     Bucket clear(Long bucketId);
     
-    List getAllItems(Long bucketId);
+    List<Item> getAllItems(Long bucketId);
```

Comment:
do the same in all method which return list of elements

Reply:
found only one mistake of this kind